recipe/meta.yaml

```diff
@@ -1,13 +1,16 @@
 {% set name = "moab" %}
 {% set version = "5.1.0" %}
 {% set sha256 = "0371fc25d2594589e95700739f01614f097b6157fb6023ed8995e582726ca658" %}
-{% set build = 5 %}
+{% set build = 6 %}
 
 # ensure mpi is defined (needed for conda-smithy recipe-lint)
 {% set mpi = mpi or 'nompi' %}
 
-{% if mpi == 'nompi' %}
-# prioritize nompi variant via build number
+# ensure tempest is defined (needed for conda-smithy recipe-lint)
+{% set tempest = tempest or 'notempest' %}
+
+{% if mpi == 'nompi' and tempest == 'notempest' %}
+# prioritize nompi and notempest variant via build number
 {% set build = build + 100 %}
 {% endif %}
```

Comment:
I'm favoring (with build number 106 instead of 6 in this case) the build with neither MPI nor TempestRemap over all other builds.  I don't think we will worry about relative priority of the builds with MPI vs. with TempestRemap vs. with both.  Users will have to explicitly pick a build if they want either or both.